### ISSUE TITLE
[front] feat: design and implement a comparison series

### ISSUE THREAD

**Issue by GresilleSiffle** [tournesol-app/tournesol#681]
## feature specifications

As a Tournesol user, on the `presidentielle2022` poll, I can start a comparison series, behaving like a tutorial.

**From where the comparison series is accessible?**

... describe where the starting point is in the app, and what are the conditions ... 

The implementation of the conditions should be done with another issue.

**Should the user be logged before starting a comparison series?**

Yes, to ease the development process.

**Should this component be usable by any poll?**

Yes we would like to, but this could be done in a later development.

**Key points**

- each step consists of a comparison page, with a specific message related to the current step
- (these messages should be easily configurable)
- the first two candidates are proposed randomly
- a breadcrumb indicates the current position (step) in the series and the remaining comparisons to do
- clicking on submit saves the comparison and goes to the next comparison page, and increases the step value by 1
- submitting a comparison already made doesn't increase the step value

**How to refresh  the candidates between each submission ?**

... to decide with the team ...

## technical specifications

Create a `<ComparisonSeries />` component.

This component should have two props:
- `length`, the length of the series
- `messages`, an object containing translated messages, each of them associated with a step

## remaining to-do

- [x] #706
- [x] review them with the team (reviewed with Aidan, Adrien and partially by Louis) // can be updated at any time
- [x] #707
- [x] #726
- (distinct topic) add a link in the future `presidentielle2022` home page
- (distinct topic) add a result page
